docs/upgrade-guide.md

```diff
@@ -15,6 +15,7 @@ Pixel sizes in line, icon and text layers now match their HTML/SVG counterparts.
 
 #### Views and Controllers
 
+* Viewport constraint props: `maxZoom`, `minZoom`, `maxPitch`, `minPitch` are no longer supported by the React component. They must be specified as part of the `viewState` object.
```

Comment:
Looks good. Maybe to further emphasize what is React specific (to let non-React users quickly skip reading it), we could write:

```
* (React only) Top level viewport constraint props removed: `DeckGL.maxZoom`, `DeckGL.minZoom`, `DeckGL.maxPitch`, `DeckGL.minPitch` props are no longer supported by the `DeckGL` React component. They must be specified as part of the `viewState` object.
* (React only) The `ViewportController` ...
```